ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/property.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+
+option java_outer_classname = "PropertyOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+message Property {
+  // Required
+  // We follow K8s' convention:
+  // '''
+  // The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
```

Comment:
I'd suggest that we adopt the prefix standard that k8s uses, as that allow different components to attach their own specific labels to a user or party, cf.: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
Ok.

For the record, adopting K8s style `ObjectMeta` message (more precisely: for now only parts of this message as used in K8s)